zerver/lib/import_realm.py

```diff
@@ -1213,6 +1213,10 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
         do_change_plan_type(realm, Realm.LIMITED, acting_user=None)
     else:
         do_change_plan_type(realm, Realm.SELF_HOSTED, acting_user=None)
+
+    RealmAuditLog.objects.create(
+        realm=realm, event_type=RealmAuditLog.REALM_IMPORTED, event_time=timezone_now()
+    )
```

Comment:
I think this probably belongs before `do_change_plan_type` -- since if that does a `RealmAuditLog` entry, it'll be clear that happened on the new server, not the old.

Reply:
It could be nice to have `extra_data` for this field that records the Zulip server version and feature level?  

Thought perhaps it'd be smarter to try to add that as actual columns in `RealmAuditLog`?  

@mateuszmandera FYI -- do you think that data in RealmAuditLog entries could be helpful in the future when writing migrations?  I guess we already have migration level data from Django's tables, so maybe this would be useless.

(In any case, this idea shouldn't block this PR).

Reply:
@timabbott Hmm I think that could be valuable - e.g. if we find a bug where some code was doing something wrong when making RealmAuditLog-logged changes and we need to migrate the affected objects to have correct data - this could allow a straightfoward way of finding the set of potentially affected objects.